src/game_io/game_loader.cc

```diff
@@ -69,76 +69,80 @@ int32_t GameLoader::load_game(bool const multiplayer) {
 		   (boost::format(_("Loading game: %1$s (%2$u/%3$d)")) % text % step % 6).str()));
 	};
 	set_progress_message(_("Elemental data"), 1);
-	log("Game: Reading Preload Data ... ");
+	log_info_time(game_.get_gametime(), "Game: Reading Preload Data ... ");
 	{
 		GamePreloadPacket p;
 		p.read(fs_, game_);
 	}
-	log("took %ums\n", timer.ms_since_last_query());
+	log_info_time(game_.get_gametime(), "took %ums\n", timer.ms_since_last_query());
```

Comment:
Looks like this:
```
[Info    @ 0:0:0.0] Reading Elemental Data ... [Info    @ 0:0:0.0] took 7ms
```
The second `[Info    @ 0:0:0.0]` needs to go, and the map and game loaders need to log without time.